ArduCopter/ArduCopter.cpp

```diff
@@ -100,7 +101,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(update_altitude,       10,    100),
     SCHED_TASK(run_nav_updates,       50,    100),
     SCHED_TASK(update_throttle_hover,100,     90),
-    SCHED_TASK(three_hz_loop,          3,     75),
```

Comment:
This isl only due to my changes. The method that I added can sometimes take up to 100us, so that's how much i added.